src/lib/Quiz.jsx

```diff
@@ -26,28 +78,41 @@ const Quiz = function ({
   }, []);
 
   useEffect(() => {
+    // declaring a common variale for all operation
+    let newQuestions = quiz.questions;
+
+    // shuffling questions  if shuffle is true
```

Comment:
remove these comments - already self-explanatory.

Reply:
ok got it